app/services/github/GitHubModule.scala

```diff
@@ -13,7 +11,7 @@ class GitHubModule extends Module {
     val organisation = configuration.underlying.getString("github.organisation")
     val gitHubApiUrl = configuration.underlying.getString("github.apiUrl")
 
-    configuration.getString("github.access.token") match {
+    configuration.getOptional[String]("github.access.token") match {
```

Comment:
Maybe we should move to `get[Option[String]]` and default to `null` for this configuration in the config file? We want to promote the use default configuration values where we can.